src/borg/repository.py

```diff
@@ -493,9 +493,23 @@ def open(self, path, exclusive, lock_wait=None, lock=True):
         self.id = unhexlify(self.config.get("repository", "id").strip())
         self.io = LoggedIO(self.path, self.max_segment_size, self.segments_per_dir)
 
+    def _load_hints(self):
+        if not (transaction_id := self.get_transaction_id()):
+            # self is a fresh repo, so transaction_id is None and there is no hints file
+            self.storage_quota_use = 0
```

Comment:
from `Repository.__init__`:
```
        self.storage_quota = storage_quota  # from param, defaults to None
        self.storage_quota_use = 0
```
So, looks like it is 0 anyway as long as nothing wrote a different value there.